pkg/grpc/port.go

```diff
@@ -0,0 +1,17 @@
+package grpc
+
+import "net"
+
+func GetFreePort() (int, error) {
```

Comment:
Could we use freeport.GetFreePort() which is getting already used in Dapr and get rid of this file?

Reply:
It's used in tests, not in `daprd`. I want to limit the amount of external dependencies as much as possible for `daprd`.